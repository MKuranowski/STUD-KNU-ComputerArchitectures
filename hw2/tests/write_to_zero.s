addi x0, x0, 42
