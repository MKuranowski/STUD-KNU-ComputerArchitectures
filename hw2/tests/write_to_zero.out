Processor's clock cycles: 1
>>>>>>>>[REGISTER DUMP]<<<<<<<
PC: = 4
x00 = 0
x01 = 0
x02 = 0
x03 = 0
x04 = 0
x05 = 0
x06 = 0
x07 = 0
x08 = 0
x09 = 0
x10 = 0
x11 = 0
x12 = 0
x13 = 0
x14 = 0
x15 = 0
x16 = 0
x17 = 0
x18 = 0
x19 = 0
x20 = 0
x21 = 0
x22 = 0
x23 = 0
x24 = 0
x25 = 0
x26 = 0
x27 = 0
x28 = 0
x29 = 0
x30 = 0
x31 = 0
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
>>>>>>>>[MEMORY DUMP]<<<<<<<<<
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
